### ISSUE TITLE
Add-Email Fails 

### ISSUE THREAD

**Issue by johannes-darms** [scheibler/khard#54]
```
Khard: Add email address to contact
Email address: test@example.com
Contact's name [test]: 
Contact test does not exist. Do you want to create it (y/n)? y
Available address books: family, friends, default
Address book [default]:  
First name: jo
Last name: jo
Organization: jo
Traceback (most recent call last):
  File "/usr/local/bin/khard", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/khard/khard.py", line 378, in main
    last_name.decode("utf-8"), organization.decode("utf-8"))
TypeError: set_name_and_organisation() takes exactly 6 arguments (4 given)
Press any key to continue...
```

Khard version 0.6.3

**Comment by scheibler** [comment]
That's a bug, which was fixed in khard version 0.7.0. Please update and try again. Current stable
version is 0.8.1.

**Comment by johannes-darms** [comment]
Thanks. Sorry for the long delay, upgrading to the newest version fixed the error.
